Products/Zuul/routers/mibs.py

```diff
@@ -256,11 +257,25 @@ def addOidMapping(self, uid, id, oid, nodetype='node'):
         if not Zuul.checkPermission('Manage DMD'):
             return DirectResponse.fail("You don't have permission to execute this command", sticky=False)
 
+        # this step is skipped during the second call when the user confirms the change
+        if check_for_conflicts:
+            organizer = self.api.getMibModuleName(uid)
+            if organizer is not None:
+                oid_list = self.api.getOidList(oid)
+                for oid_obj in oid_list:
+                    if oid_obj.moduleName != organizer:
+                        oid_data = {'old_oid': oid_obj.oid,
+                                    'old_id': oid_obj.id,
+                                    'old_moduleName': oid_obj.moduleName,
+                                    'new_moduleName': organizer,
+                                    'conflict_detected': True}
+                        return DirectResponse.succeed(data=Zuul.marshal(oid_data))
```

Comment:
This code block that executes when `check_for_conflicts` is `True` is a candidate for refactoring into its own method since its repeated in the `addTrap` method.